### PR TITLE
Test on k8s v1.19.0

### PR DESCRIPTION
**Pull request by JamieMagee**

### CODE REVIEW

**Review by viceice — CHANGES_REQUESTED** [review]
We need push branches for renovate, otherwise automerge will it work nicely

**Comment by JamieMagee** [comment]
> We need push branches for renovate, otherwise automerge will it work nicely

Is it not enough to have it just on PRs? Renovate always merges by PR in this repo.

**Comment by viceice** [comment]
Then we need to remove prCreation, so PR is created immediately, otherwise renovate will wait 24h for succeed ci checks before opening PR.

**Comment by JamieMagee** [comment]
Okay, I will re-instate the tests

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -7,8 +7,6 @@ on:
     paths:
       - 'charts/**'
```
Comment (viceice) [inline .github/workflows/ci.yaml:8]:
```suggestion
```

File: .github/kubeval.sh [hunk 2]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -euo pipefail
```
Comment (viceice) [inline .github/kubeval.sh:2]: Why this change?

**Pull request merged**